src/store/mutators.ts

```
@@ -637,7 +640,43 @@ export const updateLocalization = (
-      // adding localization
+      if (autoTranslated) {
+        // first get our existing auto translations for this uuid if we have any
+        let autoTranslations = [];
+
+        if (
+          newDef &&
+          newDef.localization &&
+          newDef.localization[language] &&
+          newDef.localization[language][uuid] &&
+          newDef.localization[language][uuid]._ui &&
+          newDef.localization[language][uuid]._ui.auto_translations
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)